src/test/regress/output/tablespace.source

```diff
@@ -251,6 +251,15 @@ ALTER TABLE ALL IN TABLESPACE testspace_renamed SET TABLESPACE pg_default;
 NOTICE:  no matching relations in tablespace "testspace_renamed" found
 -- Should succeed
 DROP TABLESPACE testspace_renamed;
+-- create a new tablespace and check that within this tablespace,
+-- after dropping the AO table, size of all deleted (but still opened) files is 0
+CREATE TABLESPACE testspace LOCATION '@testtablespace@';
+create table t(b int) with (appendonly = true) TABLESPACE testspace;
```

Comment:
I suggest to add a test for column-oriented tables

Reply:
done